src/amber/cli/templates/migration.cr

```diff
@@ -14,12 +14,12 @@ module Amber::CLI
     def initialize(@name, fields)
       @fields = fields.map { |field| Field.new(field) }
       @database = database
-      @timestamp = Time.now.to_s("%Y%m%d%H%M%S")
+      @timestamp = Time.now.to_s("%Y%m%d%H%M%S%L")
```

Comment:
Ran into an issue where the migrations had the same timestamps because crystal was so fast to generate them. All test files generated within the same second. `%L` adds milliseconds, cause thats how we roll!